drupal/rootfs/etc/cont-init.d/99-custom-setup.sh

```diff
@@ -142,8 +142,12 @@ function main {
     exit 1
   fi
 
+
+  # Rename the default drupal admin user
+  drush sql-query "UPDATE users_field_data SET name='jhu_admin' WHERE uid=1";
+
   # Set the admin password based on env vars
-  drush -l "${site_url}" user-password admin "${DRUPAL_DEFAULT_ACCOUNT_PASSWORD}";
+  drush -l "${site_url}" user-password jhu_admin "${DRUPAL_DEFAULT_ACCOUNT_PASSWORD}";
```

Comment:
I wonder if we should pull this in via an environment variable, or maybe that's what you're wondering, @little9, in your question to @jhujasonw about how private this should be.  At the very least changing this here would protect us from random attacks, but this wouldn't help with a directed attack.